docs/UserGuide.md

```diff
@@ -91,6 +96,9 @@ e.g. typing `help` and pressing Enter will open the help window. <br>
 
```

Comment:
Should we change the first glance and quick start stuff to NUSEatWhere instead of Food Guide?